plugin/helpers_plugin.go

```diff
@@ -54,3 +54,26 @@ func (p *HelpersImpl) ensureServerVersion(required string) error {
 	}
 	return nil
 }
+
+// GetPluginAssetURL builds a URL to the given asset in the assets directory.
```

Comment:
```suggestion
// GetPluginAssetURL implements GetPluginAssetURL.
```

Reply:
If we go with this comment, I'd suggest omitting it altogether, since it's of no use to the developer. I'm personally ok with "duplicating" comments like this.

Reply:
There has been issues in the past where the comment on the interface and the implementation diverged which is a situation I want to avoid in the future.

I'm 2/5 that having this comment is better then having non because it makes clear that the comment was not forgotten but instead lives somewhere else.

Reply:
@lieut-data @hanzei What's the final resolution on this?

Reply:
Let's adopt @hanzei's proposal here and be consistent with the rest of the helpers -- thanks for the discussion :)